crates/hc_sandbox/src/save.rs

```diff
@@ -64,7 +64,12 @@ pub fn clean(mut hc_dir: PathBuf, sandboxes: Vec<usize>) -> anyhow::Result<()> {
         }
         hc_dir.push(".hc");
         if hc_dir.exists() {
-            std::fs::remove_file(hc_dir)?;
+            std::fs::remove_file(&hc_dir)?;
+        }
+        hc_dir.pop();
+        hc_dir.push(".hc_auth");
```

Comment:
The auth file is automatically removed to be tidy. If the auth is being used with a sandbox then the auth is irrelevant anyway, a bit of care is needed from the user if the auth is being used against a production conductor.